android/titanium/src/java/org/appcelerator/titanium/TiApplication.java

```diff
@@ -1038,7 +992,6 @@ public void dispose()
 	{
 		TiActivityWindows.dispose();
 		TiActivitySupportHelpers.dispose();
-		TiFileHelper.getInstance().destroyTempFiles();
```

Comment:
why not destroy temp files here now? I assume because this no longer an active API and this just happens automatically implicitly?

Reply:
I've changed `TiFileHelper` to use the same temp directory we expose in JavaScript. This temp directory gets deleted on app startup and every time the JS runtime terminates. See the `deleteTiTempFiles()` method calls below...
https://github.com/appcelerator/titanium_mobile/blob/master/android/titanium/src/java/org/appcelerator/titanium/TiApplication.java#L363-L384

Also, the old `TiFileHelper` code wasn't deleting temp files if the app was forced-quit, which happens all the time while the app is backgrounded by the Android OS. This needed to be fixed.